src/components/WizardModal/_wizard-modal.scss

```diff
@@ -7,7 +7,7 @@
 
   .#{$prefix}--progress {
     /* need to pad some space for the focus outlines*/
-    padding: 3px;
+    padding: $spacing-09;
```

Comment:
$spacing-09 is 3rem. Looking at this further though, this entire style is not needed. The focus style is not 3px wide nor does it add up to 3px. We can remove this altogether.
```suggestion
```